components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml

```diff
@@ -1957,27 +1957,27 @@ spec:
           - name: AWS_ACCESS_KEY_ID
             valueFrom:
               secretKeyRef:
-                name: s3-conf
+                name: tekton-results-s3
```

Comment:
You need to change the ExternalSecret to create `tekton-results-s3` instead of `s3-conf` in tekton-logging NS: https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml#L1855

Reply:
It does that https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml#L1868 ... unless I'm missing something